server/game/stateWatchers/CardsEnteredPlayThisPhaseWatcher.ts

```diff
@@ -0,0 +1,53 @@
+import { StateWatcher } from '../core/stateWatcher/StateWatcher';
+import { StateWatcherName } from '../core/Constants';
+import { StateWatcherRegistrar } from '../core/stateWatcher/StateWatcherRegistrar';
+import Player from '../core/Player';
+import { EnteredPlayCard, PlayableCard } from '../core/card/CardTypes';
+import Game from '../core/Game';
+import { Card } from '../core/card/Card';
+
+export interface EnteredCardEntry {
+    card: EnteredPlayCard,
```

Comment:
Should be `InPlayCard` (see above comment in CardTypes.ts)

Reply:
Changed EnteredPlayCard into InPlayCard.